docs/minikube.md

```diff
@@ -0,0 +1,74 @@
+## Minikube
+
+Draft requires a wildcard domain and an ingress controller. Here how to set everything
+up when using [Minikube][Minikube]
```

Comment:
Yeah thats what I'm reading from and even from there the links don't work for me. I've seen the syntax before and it usually works fine, so now I'm really curious. Maybe I'm missing something!